project/src/test/java/com/google/sps/data/UserFeedbackTest.java

```diff
@@ -32,50 +32,39 @@
       ImmutableList.of(PLACE_ID_1, "11111", "22222");
 
   @Test
-  public void build_invalidBothChoosePlaceAndTryagain_throwsIllegalArgumentException() {
+  public void build_choosePlaceNotInRecommendedPlaces_throwsIllegalArgumentException() {
```

Comment:
chosenPlace? Same for the second test.

Reply:
Notice I wrote *chosen*, in here it says choose. I'm approving this PR but please fix before you push the changes.

Reply:
it was changed, you don't see it in the main screen because of Github - in the main screen you can't see the the new code, you just have an "outdated" mark at the top of the change request.